emcc.py

```diff
@@ -3412,22 +3409,20 @@ def modularize():
   logger.debug('Modularizing, assigning to var ' + shared.Settings.EXPORT_NAME)
   src = open(final).read()
 
-  # TODO: exports object generation for MINIMAL_RUNTIME. As a temp measure, multithreaded MINIMAL_RUNTIME builds export like regular
-  # runtime does, so that worker.js can see the JS module contents.
-  exports_object = '{}' if shared.Settings.MINIMAL_RUNTIME and not shared.Settings.USE_PTHREADS else shared.Settings.EXPORT_NAME
-
   src = '''
 function(%(EXPORT_NAME)s) {
+  var returned_promise_resolve, returned_promise_reject;
+  var promise = new Promise(function(resolve, reject) { returned_promise_resolve=resolve, returned_promise_reject=reject; });
```

Comment:
please put the function body on another line,
```javascript
var x = new Promise(function(..) {
  ..
});
```

Reply:
Done